src/components/atoms/Card/Card.tsx

```diff
@@ -39,12 +39,13 @@ const SCard = styled.div<ICardProps>`
   `};
 `;
 
-const Card: React.FunctionComponent<ICardProps> = props => {
-  return <SCard {...props} />;
-};
-
 Card.defaultProps = {
   type: 'card',
 };
 
+// TODO: Styleguidist to be able to locate styled components. See #147.
```

Comment:
@poteirard I was trying to do the following as per our conversation:
```ts
interface IStyledComponentProps {
  as?: string;
  forwardedAs?: string;
  css?: string;
}

export interface ICardProps extends HTMLAttributes<HTMLDivElement>, IStyledComponentProps {
  type?: TCardTypes;
  display?: 'block' | 'none' | 'inline-block';
}
```
However given we spread props at the bottom **TS** complains about types mismatch:
 
![Screenshot 2019-12-10 at 11 17 02](https://user-images.githubusercontent.com/5938217/70520926-a2b64d00-1b3e-11ea-9f57-8437b2efa5f7.png)

I'm unsure whether there's a straightforward solution to this, any ideas?